packages/hooks/src/utils/check.ts

```diff
@@ -0,0 +1,10 @@
+export function devCheckDecorator(fn: Function) {
+  return function () {
+    if (process.env.NODE_ENV === 'development') {
+      if (typeof arguments[0] !== 'function') {
+        console.error(`${fn.name} expected parameter is a function, got '${typeof fn}`);
+      }
+    }
+    return fn.apply(this, arguments);
+  };
```

Comment:
```suggestion
  if (process.env.NODE_ENV === 'development') {
    return function () {
      if (typeof arguments[0] !== 'function') {
        console.error(`${fn.name} expected parameter is a function, got '${typeof fn}`);
      }
      return fn.apply(this, arguments);
    };
  } else {
    return fn
  }
```

`production` 模式下就不要再多包一层了吧……整段逻辑都跳过

Reply:
以及，这个高阶函数的命名感觉不太合适呀……